spec/requests/atmosphere/api/clew_spec.rb

```diff
@@ -172,20 +167,29 @@ def cs
         expect(clew_at_response['appliance_types'][1]['matched_flavor']).to clew_flavor_eq  flavor
 
         #TODO complete this test to check all the returned fields
-
       end
+    end
+
+    it 'does not return AT with visible_to owner when user is not owner' do
```

Comment:
This is a general solution for all `index` like rest services. If you want to use admin abilities you need to add `all=true` param. Take a look at https://github.com/dice-cyfronet/atmosphere/blob/master/app/controllers/atmosphere/api/application_controller.rb#L84-L86 for details.